packages/pluggableWidgets/line-chart-native/src/components/__tests__/__snapshots__/Legend.spec.tsx.snap

```diff
@@ -0,0 +1,78 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Legend doesn't render series when it cannot retrieve a background color from the series style property 1`] = `
+<View>
```

Comment:
Output is just views, shouldnt this be SVG?

Reply:
Legend isn't part of the lib, so that's why it's views. This will be the case for labels in the future as well (in that case `Text`).